src/Health/Tests/Endpoints.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Tribe\Extensions\Promoter\Health\Tests;
+
+use Tribe\Extensions\Promoter\Health\Critical_Exception;
+use Tribe__Promoter__Connector;
+
+class Endpoints implements Runnable {
+
+	public function run() {
+		$this->connector = tribe( 'promoter.connector' );
+
+		if ( ! $this->connector instanceof Tribe__Promoter__Connector ) {
+			throw new Critical_Exception( __( 'The connector class is not defined.', 'tribe_extensions_promoter' ) );
+		}
+
+		$endpoints = [
+			get_rest_url( get_current_blog_id(), 'tribe/events/v1/events' ),
+			get_rest_url( get_current_blog_id(), 'tribe/tickets/v1/tickets' ),
+		];
+
+		foreach ( $endpoints as $endpoint ) {
+			$response = wp_remote_get( $endpoint, [
+				'timeout'   => 30,
+				'sslverify' => false,
```

Comment:
Why? Add a compelling explanation if needing to stray from the default

Reply:
As the connector can be set to a local URL which means in that case the SSL is not required to be verified + to follow the same pattern that has been applied in common to this same logic.

Reply:
makes sense to add a comment